includes/discovery/mempools/netonix.inc.php

```diff
@@ -1,22 +1,34 @@
 <?php
-  /*
-   * LibreNMS module for Netonix
-   *
-   * Copyright (c) 2016 Tony Murray <murraytony@gmail.com>
-   *
-   * This program is free software: you can redistribute it and/or modify it
-   * under the terms of the GNU General Public License as published by the
-   * Free Software Foundation, either version 3 of the License, or (at your
-   * option) any later version.  Please see LICENSE.txt at the top level of
-   * the source code distribution for details.
-   */
+/**
+ * netonix.inc.php
+ *
+ * LibreNMS mempools module for Netonix
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2016 Tony Murray
+ * @author     Tony Murray <murraytony@gmail.com>
+ */
 
 if ($device['os'] == 'netonix') {
     echo 'NETONIX : ';
 
-    $free = str_replace('"', "", snmp_get($device, 'UCD-SNMP-MIB::memTotalFree.0', '-OvQU'));
+    $free = snmp_get($device, 'UCD-SNMP-MIB::memTotalFree.0', '-OvQU');
```

Comment:
It wasn't needed on my device machine, I'll test on another device.  I think it was just copy/paste before...